lua/damagelogs/config/config.lua

```diff
@@ -49,36 +50,36 @@ REQUIRES ULX ! If you are using ServerGuard, set this to 0 (it will use ServerGu
 ]]
 --
 Damagelog.ULX_AutoslayMode = 1
--- Force autoslain players to be innocents (ULX only)
+-- Force autoslain players to be innocents (ULX/SAM only)
 -- Do not enable this if another addon interferes with roles (Pointshop roles for example)
 Damagelog.ULX_Autoslay_ForceRole = true
--- Default autoslay reasons (ULX and ServerGuard)
-Damagelog.Autoslay_DefaultReason = "broke server rules"
-Damagelog.Autoslay_DefaultReason1 = "random kill"
-Damagelog.Autoslay_DefaultReason2 = "multiple random kills"
-Damagelog.Autoslay_DefaultReason3 = "random damage"
-Damagelog.Autoslay_DefaultReason4 = "multiple random damage"
-Damagelog.Autoslay_DefaultReason5 = "teamkill"
-Damagelog.Autoslay_DefaultReason6 = "needless report"
-Damagelog.Autoslay_DefaultReason7 = "unfitting answer"
-Damagelog.Autoslay_DefaultReason8 = "unfitting language"
-Damagelog.Autoslay_DefaultReason9 = "lying"
-Damagelog.Autoslay_DefaultReason10 = "propkill"
-Damagelog.Autoslay_DefaultReason11 = "teaming"
-Damagelog.Autoslay_DefaultReason12 = "random kos"
+-- Default autoslay reasons (ULX, SAM, and ServerGuard)
+Damagelog.Autoslay_DefaultReason1 = "Random Damage"
+Damagelog.Autoslay_DefaultReason2 = "RDM"
+Damagelog.Autoslay_DefaultReason3 = "2x RDM"
+Damagelog.Autoslay_DefaultReason4 = "Attempted Mass"
+Damagelog.Autoslay_DefaultReason5 = "Mass RDM"
+Damagelog.Autoslay_DefaultReason6 = "Super Mass"
+Damagelog.Autoslay_DefaultReason7 = "Ghosting"
+Damagelog.Autoslay_DefaultReason8 = "Hacking"
+Damagelog.Autoslay_DefaultReason9 = "Prop kill"
+Damagelog.Autoslay_DefaultReason10 = "Consistent RDM"
+Damagelog.Autoslay_DefaultReason11 = "Yolo"
+Damagelog.Autoslay_DefaultReason12 = "Yolo"
+
 -- Default ban reasons (ULX and ServerGuard)
-Damagelog.Ban_DefaultReason1 = "random kill"
-Damagelog.Ban_DefaultReason2 = "multiple random kills"
-Damagelog.Ban_DefaultReason3 = "random damage"
-Damagelog.Ban_DefaultReason4 = "multiple random damage"
-Damagelog.Ban_DefaultReason5 = "unfitting answer"
-Damagelog.Ban_DefaultReason6 = "unfitting answers"
-Damagelog.Ban_DefaultReason7 = "unfitting language"
-Damagelog.Ban_DefaultReason8 = "teaming"
-Damagelog.Ban_DefaultReason9 = "ghosting"
-Damagelog.Ban_DefaultReason10 = "rude"
-Damagelog.Ban_DefaultReason11 = "cheating"
-Damagelog.Ban_DefaultReason12 = "spam"
+Damagelog.Ban_DefaultReason1 = "Random Damage and leave"
+Damagelog.Ban_DefaultReason2 = "RDM and leave"
+Damagelog.Ban_DefaultReason3 = "2x RDM and leave"
+Damagelog.Ban_DefaultReason4 = "Attempted Mass and leave"
+Damagelog.Ban_DefaultReason5 = "Mass RDM"
+Damagelog.Ban_DefaultReason6 = "Super Mass"
+Damagelog.Ban_DefaultReason7 = "Ghosting"
+Damagelog.Ban_DefaultReason8 = "Hacking"
+Damagelog.Ban_DefaultReason9 = "Consistent RDM"
+Damagelog.Ban_DefaultReason10 = "Yolo"
+Damagelog.Ban_DefaultReason11 = "Yolo"
+Damagelog.Ban_DefaultReason12 = "Yolo"
```

Comment:
Could the Yolos be replaced with something else?

Reply:
done